task-ballerina/build.gradle

```diff
@@ -99,16 +101,20 @@ task copyStdlibs(type: Copy) {
 task updateTomlVersions {
     inputs.dir file(project.projectDir)
     doLast {
+        def stdlibDependentTimeVersion = project.stdlibTimeVersion.split("-")[0]
         def newConfig = ballerinaConfigFile.text.replace("@project.version@", project.version)
         newConfig = newConfig.replace("@toml.version@", tomlVersion)
         newConfig = newConfig.replace("@quartz.version@", quartzVersion)
         ballerinaConfigFile.text = newConfig
+        def newDependencyConfig = ballerinaDependencyFile.text.replace("@stdlib.time.version@", stdlibDependentTimeVersion)
```

Comment:
Looks like you've added the logic to update the time package version at the build time, but did not add the place holder to this to work.